pkgs/tools/security/fprintd/default.nix

```
@@ -1,42 +1,50 @@
-{ thinkpad ? false
-, stdenv
-, fetchurl
-, fetchpatch
+{ stdenv
+, fetchFromGitLab
 , pkgconfig
+, meson
+, ninja
 , intltool
-, libfprint-thinkpad ? null
-, libfprint ? null
+, libpam-wrapper
+, cairo
+, gtk-doc
 , glib
 , dbus-glib
 , polkit
 , nss
 , pam
 , systemd
-, autoreconfHook
-, gtk-doc
+, libfprint
+, python37Packages
 }:
 
 stdenv.mkDerivation rec {
-  pname = "fprintd" + stdenv.lib.optionalString thinkpad "-thinkpad";
-  version = "0.9.0";
+  pname = "fprintd";
+  version = "1.90.1";
 
```

gtk-doc will not be persisted without devdoc output:

```suggestion

  outputs = [ "out" "devdoc" ];

```